static/src/javascripts/projects/common/modules/commercial/sticky-top-banner.js

```diff
@@ -211,7 +211,9 @@ define([
     };
 
     var initialise = function () {
-        if (detect.isBreakpoint({ min: 'desktop' })) {
+        // Although we check as much config as possible to decide whether to run sticky-top-banner,
+        // it is still entirely possible for the ad slot to be closed.
+        if (detect.isBreakpoint({ min: 'desktop' }) && $adBannerInner[0]) {
```

Comment:
Hopefully this is the catch-all line which stops all the errors flooding sentry. But the rest of the PR is about making sure the ad slot really is there.